_activities/gsdocs.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Season of Docs"
+author: "Javier Cervantes"
+layout: default
+---
+
+# ![CERN](/images/CERN-logo.jpg){:height="100px"} ![HSF](/images/hsf_logo_angled.png){:height="100px"} Season of Docs 2019
+
+## Introduction
+
+[Season of Docs](https://developers.google.com/season-of-docs/) is a program that brings technical writers and open source projects together for a few months to work on open source documentation.
+
+Particle physics is an exciting field where large collaborations of scientists collect
+and analyze petabytes data from high-energy physics experiments, such as the Large Hadron Collider,
+hosted at the CERN laboratory in Geneva, Switzerland.
+Some of the questions that we collectively ask are:
+
+- what are the fundamental blocks that make up our Universe?
+- what is the nature of dark matter and dark energy?
+- what is the nature of the asymmetry between matter and antimatter?
+- what was early Universe like?
+
+To answer these questions, particle physicists build software to simulate and analyze what happens in particle physics detectors.
+
+Since 2011, CERN has participated in other Google initiatives such as Google Summer of Code (GSoC), first as a small organization (CERN-SFT) and later as an umbrella organization (CERN-HSF) to involve the high-energy physics community. Given the good results in the past under the GSoC program, CERN-HSF is eager to participate on this first edition of Season of Docs.
+
+
+### Season of Docs 2019
+
+In 2019 CERN-HSF is planning to apply as an umbrella organization, project proposals are listed [below](#projects-in-2019).
+
+
+## Projects in 2019
+
+{% assign current_year = "2019" %}
+{% include gsdocs_project_list.ext year=current_year %}
+
+## Participating Organizations in 2019
+
+{% include gsdocs_organization_list.ext year=current_year %}
+
+## Summary
+
+[Full list of Proposal Ideas](/gsdocs/2019/summary.html)
+
+[Full list of Mentors](/gsdocs/2019/mentors.html)
+
+---
+
+*HSF Season of Docs Administrators*: Andrei Gheata and Javier Cervantes Villanueva.
+
+[Contact us](mailto:hsf-gsdocs-admin@googlegroups.com)
+[General e-mail](mailto:hep-software-foundation-google-summer-of-code@googlegroups.com)
```

Comment:
we need to change also this mailing list

Reply:
Right, should we create the mailing lists proposed above?

Reply:
@agheata ping